### PR TITLE
MentionAvatars: Add option to hide @ symbol

### PR DESCRIPTION
**Pull request by Luna-devv**
This adds an option to the Mention Avatars plugin to remove the `@` symbol.
image

### CODE REVIEW

**Comment by Nuckyz** [comment]
move the settings definition to the index file

File: src/plugins/mentionAvatars/index.tsx [hunk 1]
```diff
@@ -6,16 +6,25 @@
...
+import definePlugin, { OptionType } from "@utils/types";
 import { SelectedGuildStore, useState } from "@webpack/common";
 import { User } from "discord-types/general";
 
+const settings = definePluginSettings({
+    hideAtSymbol: {
+        type: OptionType.BOOLEAN,
+        description: "Whether the the @-symbol should be hidden.",
+        default: false
+    }
```
Comment (Vendicated) [inline src/plugins/mentionAvatars/index.tsx:21]: generally switches should be positive

```suggestion
    showAtSymbol: {
        type: OptionType.BOOLEAN,
        description: "Whether the the @ symbol should be displayed",
        default: true
    }
```

File: src/plugins/mentionAvatars/index.tsx [hunk 2]
```diff
@@ -25,20 +34,28 @@ export default definePlugin({
...
+                {getUsernameString(username)}
             </span>
         );
     }, { noop: true })
+
 });
+
+function getUsernameString(username: string) {
+    if (settings.store.hideAtSymbol) return username;
+    return `@${username}`;
```
Comment (Vendicated) [inline src/plugins/mentionAvatars/index.tsx:60]:
```suggestion
    return settings.store.hideAtSymbol
    	? username;
    	: `@${username}`;
```

File: src/plugins/mentionAvatars/index.tsx [hunk 3]
```diff
@@ -6,16 +6,25 @@
...
+        description: "Whether the the @-symbol should be hidden.",
+        default: false
+    }
+});
+
 export default definePlugin({
     name: "MentionAvatars",
     description: "Shows user avatars inside mentions",
-    authors: [Devs.Ven],
+    authors: [Devs.Ven, Devs.Luna],
```
Comment (Vendicated) [inline src/plugins/mentionAvatars/index.tsx:27]: please don't add yourself for such minor changes

```suggestion
    authors: [Devs.Ven],
```

**Comment by Vendicated** [comment]
thanks!

**Pull request merged**